.eslintrc.js

```diff
@@ -117,6 +117,12 @@ module.exports = {
                 'compat/compat': 'off',
             },
         },
+        {
+            files: ['react/**/*.ts'],
+            rules: {
+                'posthog-js/no-direct-null-check': 'off',
```

Comment:
why turn this off instead of using `isNull`?

Reply:
Side note, but what's the background on us using `isNull` etc rather than `===`? Browser compatibility? Bytes?

Reply:
the bundle reduced by 1% when we switched to `isNull` and `isUndefined` compared to the straight comparison

cos `fe(x, y)` is fewer characters than `x === y` and we use it a lot

ofc likely that the react stuff is being bundled and run through a minifier by someone else but consistency is probably better than varying

Reply:
we use the checks a lot so the minified code has (something like) `fe(x)` at each call site 

whereas the direct check is (something like) `x === void 0` when minified 

so once you call `isUndefined` more times than the character count of adding the declaration to the bundle you  add roughly half as many characters by moving to a function

it's the same reason we have some global things as variables in the `utils` folder because then they'll minify and we have `xY` in the bundle not `navigator`

Reply:
Hmm, in my mind I did that to avoid crossing the boundary from `posthog-js/react` to `posthog-js`. I now see that doesn't make sense because this is a single package for those who're installing it, so I'm happy to change :)

Reply:
Ah.. fair.. yep, something to bear in my mind if/when we start splitting this up

Reply:
> So if someone was experimenting with a new minifier, this is something they could change to see if it made a difference.

that'd be a very good minifier... it's more of a compiler at that point 🤯

Reply:
there's two sources of size in the repo

* unnecessary repetition of primitives - like this example
* heavy use of classes and no minification of class fields/properties